.github/workflows/gpl-linux-arm64.yml

```diff
@@ -0,0 +1,64 @@
+name: GPL Linux ARM64 release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    if: (github.repository == 'cvc5/cvc5')
+    strategy:
+      matrix:
+        include:
+          - name: shared
+            build-opts: --java-bindings
+
+          - name: static
+            build-opts: --static
+    
+    name: ${{ matrix.name }}
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+      with:
+        platforms: "linux/arm64"
+
+    - name: Build cvc5
+      uses: docker/build-push-action@v5
+      with:
+        file: contrib/arm64-gpl.Dockerfile
+        context: ./
+        load: true
+        tags: cvc5-arm64-${{ matrix.name }}-gpl:latest
+        build-args: |
+          OTHER_OPTS=${{ matrix.build-opts }}
+
+    - name: Extract install directory
+      id: extract
+      uses: shrink/actions-docker-extract@v3
+      with:
+        image: cvc5-arm64-${{ matrix.name }}-gpl:latest
+        path: /cvc5/install
+        destination: ./
+
+    - name: Create ZIP file
+      id: create_zip
+      run: |
+        PACKAGE_NAME=cvc5-Linux-arm64-${{ matrix.name }}-gpl
+        mv install $PACKAGE_NAME
+        zip -r $PACKAGE_NAME $PACKAGE_NAME
+        echo "filepath=./$PACKAGE_NAME.zip" >> $GITHUB_OUTPUT
+        #echo "release=latest" >> $GITHUB_OUTPUT
```

Comment:
Is this commented out on purpose? Should the line be deleted?

Reply:
It was for a test. I deleted the line.